src/main/java/com/google/api/gax/grpc/InstantiatingChannelProvider.java

```diff
@@ -196,9 +218,8 @@ public static Builder newBuilder() {
 
   public static final class Builder {
 
-    // Default names and versions of the client and the service generator.
+    // Default names and versions of the service generator.
     private static final String DEFAULT_GENERATOR_NAME = "gapic";
-    private static final String DEFAULT_CLIENT_LIB_NAME = "gax";
     private static final String DEFAULT_GEN_VERSION = "0.1.0";
```

Comment:
@shinfan @garrettjonesgoogle From a metrics perspective, I would prefer that this be `"UNKNOWN"`, so that it doesn't look like an actual version number. It will be really difficult to analyze the data if sometimes we use `"UNKNOWN"` and other times we use some other value like `"0.1.0"`.